.gitignore

```diff
@@ -44,3 +42,9 @@ site/
 # helm dependencies temporary folders
 codacy/charts
 codacy/tmpcharts
+
+# Release notes and changelogs automated generation
+release-notes-tools
```

Comment:
This ensures that the generated Markdown files are not committed to the public repository. :+1: